services/user_service/main.py

```diff
@@ -1,5 +1,19 @@
-import subscriber
-import publisher
-import database_connector
+import threading
+import callbacks
 
-print("User Service entry point")
+from constants import pubsub_manager
+
+subs = {
+    "users_get-sub": callbacks.get_users_callback,
+    "user_get_by_id-sub": callbacks.get_user_by_id_callback,
+    "user_get_by_email-sub": callbacks.get_user_by_email_callback,
+    "user_save-sub": callbacks.save_user_callback,
+    "user_update-sub": callbacks.update_user_callback,
+    "users_delete_all-sub": callbacks.drop_users_callback,
+    "user_delete_by_id-sub": callbacks.delete_user_by_id_callback,
+    "user_delete_by_email-sub": callbacks.delete_user_by_email_callback,
+}
+
+if __name__ == "__main__":
+    for i in subs:
+        threading.Thread(target=pubsub_manager.subscribe, args=[i, subs[i]]).start()
```

Comment:
`[threading.Thread(target=pubsub_manager.subscribe, args=a).start() for a in subs.items()]`
won't work?

Reply:
Btw, @arihilmir  is checking if threads are not ready to be sure everything works as expected
How you handle that?

Reply:
@ZhavoronkovN let's keep it simple. List comprehension is quite usefull thing, but there is no need to use it everywhere

Reply:
@ZhavoronkovN I do not know. They should be always ready.

Reply:
They should never be ready, if they are ready we are not getting any messages
For what keeping it simple? We don't write examples for book "Python for children"

Reply:
Do not agree about list comprehension. We do not pay for each line of code).

Reply:
@ZhavoronkovN sorry, I did not explain it  correctly. The thread is ready means it's alive and subscriber inside of the thread continue receiving the messages. Do we really need to explicitely check it?

Reply:
Don't know. It should be part of you investigation, do we really should

Reply:
@ZhavoronkovN it works (at least for now) without checking whether the thread is ready. Since the thread is alive until subscriber inside of the thread continue receiving the messages. If error happens we need to restart probably the service

Reply:
How we would know that error happened?

Reply:
The client library will raise an exception. This will be ok. I think, we'll configure service restart on error, so everything should be ok.

Reply:
From my experience, Python thread will just silently fall and we will lost messages for some topic

Reply:
![image](https://user-images.githubusercontent.com/37078272/146263059-fee2b47c-b3c3-4d9b-8fbd-dc8d47ed5f2d.png)
![image](https://user-images.githubusercontent.com/37078272/146263104-45520da3-8cb5-4780-b07c-2b5e8e428189.png)

Looks like @ZhavoronkovN is right

Reply:
So, you might use smth like `exit(1)` (idk) to shut down the app when a thread is down.

Or use the async approach.